apps/crn-frontend/src/shared-research/ResearchOutput.tsx

```diff
@@ -52,10 +52,12 @@ const ResearchOutput: React.FC = () => {
               researchOutputData={researchOutputData}
             />
           ) : (
-            <TeamOutput
-              teamId={researchOutputData.teams[0]?.id}
-              researchOutputData={researchOutputData}
-            />
+            researchOutputData.teams[0]?.id && (
+              <TeamOutput
+                teamId={researchOutputData.teams[0].id}
+                researchOutputData={researchOutputData}
+              />
+            )
```

Comment:
If `researchOutputData.teams[0]?.id` aren't we going to get undefined displayed in the document?

Reply:
undefineds dont get rendered

Reply:
a lot of code would be incorrect if that would be the case 😄